docs/GDI-SOP_charter.md

```diff
@@ -0,0 +1,163 @@
+
+### In-Scope
+#### Roles
+- **Helpdesk Staff and Operations Bioinformaticians**: Responsible for responding to and resolving user queries and issues within the node.
+- **Data Stewards, Data Curators, and Bioinformaticians**: Focused on the preparation, harmonization, and curation of data and metadata to ensure high-quality data management.
+- **Software Developers, Software Engineers, and DevOps Engineers**: Tasked with designing, developing, implementing, deploying, and monitoring software and services deployed at a node.
+
+#### Topics
+- **Data Protection & Security**: Includes procedures related to security incident response, risk assessment, and compliance with data protection regulations (e.g., DPA, DAA, DPIA).
+- **Data & Metadata Management**: Covers data management plans (DMPs), and processes for ensuring data quality and harmonization within the node, including interactions with the GDI User Portal.
+- **Technical Infrastructure & Software Development**: Focuses on technical certifications, compliance assessment tools, and ensuring that node infrastructure meets GDI standards.
+- **Helpdesk & Operations**: Encompasses standard operating procedures for helpdesk interactions with users, including interfacing with the GDI User Portal.
+
+#### Resources
+- **SOP Examples & Templates**: Provision of standardized templates for SOPs that can be adapted by nodes.
+- **Operational Recommendations & Guidelines**: Guidance on best practices for operating node services within the GDI network.
+- **Training on SOP Usage & Management**: Training materials and sessions to ensure that node staff can effectively implement and manage SOPs.
+- **Resource Management Plan**: A plan for the management of resources required for SOP implementation.
+
+### Out-of-scope
+#### Roles
+- **Researchers Using Node Services**: While crucial to the broader GDI network, their specific operational procedures are not covered by these SOPs.
+- **Training, Outreach, and Communications Officers**: These roles, while important, are not the focus of the current SOP scope.
+
+#### Topics
+- **Governance, Strategy & Sustainability**: These high-level topics, including aspects like GDI governance and long-term sustainability strategies, are beyond the scope of these SOPs.
+- **User Training on Node Services**: Training materials and programs designed to educate users (researchers) on how to use node services are not covered.
+- **Community Outreach & Engagement**: Activities related to building community engagement and outreach are excluded.
+
+#### Resources
+- **Node-Specific SOPs**: SOPs tailored to the unique needs of individual nodes are not included, as the focus is on providing common templates and guidelines.
```

Comment:
common templates or standard templates?

Reply:
Hmmm this I took from Mallory's presentation. I think common templates is also correct, since we are to give "common to GDI" templates across nodes. But I can change it to standard.